ooniprobe/Utility/TestUtility.mm

```diff
@@ -143,4 +143,11 @@ + (void)writeString:(NSString*)str toFile:(NSString*)fileName{
         [data writeToFile:fileName atomically:YES];
     }
 }
+
++ (NSUInteger)makeTimeout:(NSUInteger)bytes{
+    //Timeout dependent on the body size considering a minimum upload speed of 12.5 kbit/s + 10 s
```

Comment:
I think this comment is now inaccurate because we rounded. I think that's 16 Kbit/s.